src/hooks/useConcepts.tsx

```diff
@@ -39,7 +39,10 @@ export function useConcepts(references: Set<string>): {
 
   const results = useMemo(
     () => ({
-      concepts: data ? [].concat(data?.map((res) => res?.data?.results).flat()) : undefined,
+      // data?.[0] check is added for tests, as response is undefined in tests
+      // hence the returned concepts are [undefined], which breaks the form-helper.ts
+      // As it cannot read `uuid` of `undefined`
+      concepts: data && data?.[0] ? [].concat(...data.map((res) => res?.data?.results)) : undefined,
```

Comment:
```suggestion
      concepts: data?.[0] ? [].concat(data.map((res) => res?.data?.results)) : undefined,
```

Reply:
Hi @samuelmale!

`[].concat(data.map((res) => res?.data?.results))` will return [[Resp, Resp, Resp]], but we want the final array to be `[Resp, Resp, Resp]`

`Resp` represents a response set.